HubTurboUpdate.json

```diff
@@ -0,0 +1,12 @@
+{
+  "hubturboVersionsDownloadLink": [
```

Comment:
This is really just cosmetic, but can this be something shorter, like `downloadLinks`? We know that it's related to HT versions.